### PR TITLE
feat(route): Add wmpvp

### PR DESCRIPTION
**Pull request by tssujt**
## Involved Issue / 该 PR 相关 Issue

Close #

## Example for the Proposed Route(s) / 路由地址示例

```routes
/wmpvp/news/1
/wmpvp/news/2
```

## New RSS Route Checklist / 新 RSS 路由检查表
  
- [x] New Route / 新的路由
  - [x] Follows [v2 Script Standard](https://docs.rsshub.app/joinus/advanced/script-standard) / 跟随 [v2 路由规范](https://docs.rsshub.app/zh/joinus/advanced/script-standard)
- [x] Documentation / 文档说明
- [ ] Full text / 全文获取
  - [ ] Use cache / 使用缓存
- [ ] Anti-bot or rate limit / 反爬/频率限制
  - [ ] If yes, do your code reflect this sign? / 如果有, 是否有对应的措施?
- [x] [Date and time](https://docs.rsshub.app/joinus/advanced/pub-date) / [日期和时间](https://docs.rsshub.app/zh/joinus/advanced/pub-date)
  - [x] Parsed / 可以解析
  - [x] Correct time zone / 时区正确
- [ ] New package added / 添加了新的包
- [ ] `Puppeteer`

### CODE REVIEW

File: lib/v2/wmpvp/index.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        const newsId = entity.newsId;
+        const newsLink = `https://news.wmpvp.com/news.html?id=${newsId}&gameTypeStr=${type}`;
+
+        // 最终需要返回的对象
+        return {
+            title: entity.title,
+            pubDate: parseDate(entity.publishTime),
+            link: newsLink,
+            guid: newsLink,
+            image: entity.thumbnail,
```
Comment (TonyRL) [inline lib/v2/wmpvp/index.js:24]: Feed item does not have image property. 
docs: https://docs.rsshub.app/joinus/advanced/advanced-feed

Comment (tssujt) [reply lib/v2/wmpvp/index.js]: OK. Now the `image` entity has been removed.

**Review by TonyRL — COMMENTED** [review]
https://github.com/DIYgod/RSSHub/actions/runs/8079241331/job/22073230682#step:6:19

**Comment by tssujt** [comment]
@TonyRL It seems that the action of semgrep failed due to the unstable service of semgrep. Can you help me retry it?

File: lib/v2/wmpvp/radar.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+module.exports = {
+    'wmpvp.com': {
+        _name: '完美世界电竞',
+        '.': [
+            {
+                title: '资讯',
+                docs: 'https://docs.rsshub.app/routes/game#wmpvp-zi-xun',
```
Comment (TonyRL) [inline lib/v2/wmpvp/radar.js:7]:
```suggestion
                docs: 'https://docs.rsshub.app/routes/game#wan-mei-shi-jie-dian-jing',
```

Comment (tssujt) [reply lib/v2/wmpvp/radar.js]: Route has been renamed from `wmpvp-zi-xun` to `wan-mei-shi-jie-dian-jing-zi-xun`

**Comment by tssujt** [comment]
@TonyRL Thanks!

**Pull request merged**